src/rgw/rgw_website.h

```diff
@@ -192,8 +205,24 @@ struct RGWBucketWebsiteConf
   void decode_xml(XMLObj *obj);
   void dump_xml(Formatter *f) const;
 
-  bool should_redirect(const string& key, const int http_error_code, RGWBWRoutingRule *redirect);
-  void get_effective_key(const string& key, string *effective_key);
+  bool should_redirect(const std::string& key,
+                       const int http_error_code,
+                       RGWBWRoutingRule *redirect);
+
+  void get_effective_key(const std::string& key,
+                         std::string *effective_key) const;
+
+  const std::string& get_swift_index_doc() const {
+    return index_doc_suffix;
+  }
```

Comment:
it doesn't look like `index_doc_suffix` is specific to swift - can you rename to `get_index_doc()`?